notebooks/Notebook2.md

```diff
@@ -2,33 +2,36 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+This week I mostly focused on the parser combinator. I still have a bit of troubleshooting and progress to make.
+Based on feedback I also focused more on the syntax of the DSL, as the feedback mentioned it was vague/not clear
+enough. I also realized there were features in calendar apps/schedules which the DSL doesn't have and would probably
+make using it easier if it had these features.
```

Comment:
I respect that you took the feedback about vague/unclear syntax seriously. This project felt like it had a specific vision, but adding a few features and focusing on clear syntax I think will improve the project over all!